cernopendata/modules/fixtures/data/docs/cms-virtual-machine-2011/cms-virtual-machine-2011.md

```diff
@@ -12,12 +12,12 @@ VirtualBox is a free, open source and multiplatform application to run virtual m
 
 You will need administrative ("root") privileges on every platform to perform the installation of VirtualBox.
 
-Note: the latest tested version of VirtualBox working with this CMS-specific CernVM image is 5.0.14. If you have troubles with the latest version of VirtualBox, pick that one: the full history of VirtualBox versions is available [on a different page][installVB2].
+Note: the latest tested version of VirtualBox working with this CMS-specific CernVM image is 5.2.2. If you have troubles with the latest version of VirtualBox, pick that one: the full history of VirtualBox versions is available [on a different page][installVB2].
 
 
 ### Step 2: Downloading and Creating a Virtual Machine
 
-**Important**: Before you download the CernVM, note that the imported settings may not always work on your host machine. Please see [Issues and Limitations][issues] if you encounter any problems with booting the VM.
```

Comment:
this one should stay [issues] because the link is defined at the end of the page

Reply:
It went wrong (to search?), but maybe there was an intermediate fix fixing the wrong behaviour which was not propagated to dev yesterday afternoon? Changed back.

Reply:
I see now, you are right!
It should actually be (#issue) :D and I guess the line 172 can be removed 
(sorry)